### DISCUSSION TITLE
Main output compression settings...

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#255] [Ideas]
I was wondering, would it be possible to manipulate the output compression outside the mfx, somewhere in the system  ? To have the option to compress the sound and use the stereo MFX at the same time…

**Comment by tripsylon** [comment]
Bad idea?

**Reply by BKLronin** [reply]
I suppose thats not very like to do be doable. Its a hack so mostly reverse engineered. That means smaller changes within whats already there. 

I´d imagine Korg packed it already with features as much as it was possible.

**Reply by AndrewSosna** [reply]
I have a feeling that compression is not just a "setting" but an additional effect that requires resources, so I guess this request is, actually, "can we have two MFX at the same time". Maybe I'm wrong.

**Reply by tripsylon** [reply]
OK I thought that maybe there is some compression all ready going on on the output of the soundengine...

**Reply by bangcorrupt** [reply]
I haven't found anything in the code, but maybe there is some dynamics control before clipping.  I'd like to be able to decouple the sound generators from the effects, then route the output of one effect into the next.

**Reply by tripsylon** [reply]
That would be genius, if we could at lest tweak the threshold or maybe an master eq independent of other MFX !!!
